app/src/main/java/com/example/myfridgehome/dto/SetStoredFoods.kt

```diff
@@ -8,16 +8,14 @@ import java.io.OutputStream
 import java.io.OutputStreamWriter
 import java.io.StringWriter
 
-
-
 //this class collects the user input and converts it into a writable string format.
 abstract class SetStoredFoods(var name: String, var type: String, var quantity : Int, var measurement : String)
 {
     //collects input from the user and assigns to local values.
-    val nm = name
-    val ty = type
-    val qt = quantity
-    val mt = measurement
+    private val nm = name
```

Comment:
I'd do a grander refactoring.
* Two letter variables are a self-documenting no-no.  You could rename them to something more descriptive, or...
* Instead of accepting each of these values individually, in the constructor, pass in a data class that encapsulates all of them.  This is a more object oriented approach.